### PR TITLE
CHE-490: add footer with product logo and support stuff

### PR DESCRIPTION
**Pull request by ashumilova**
footer_codenvy

### CODE REVIEW

File: src/assets/branding/product.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
   "title": "Codenvy - Dashboard",
   "name": "Codenvy",
-  "logoFile": "logo@3x.png",
```
Comment (benoitf) [inline src/assets/branding/product.json]: this referenced file should be remoded in that case no ?

Comment (ashumilova) [inline src/assets/branding/product.json]: yes, fixed this

**Comment by benoitf** [comment]
+1

**Comment by olexii4** [comment]
+1

**Comment by akurinnoy** [comment]
+1

**Pull request merged**